src/styles/index.css

```diff
@@ -47,10 +47,26 @@
 
     --p-color-live: #6272FF;
 
-    --p-color-sentiment-positive: #22C55E;
-    --p-color-sentiment-neutral: #70707B;
-    --p-color-sentiment-warning: #F5A50A;
-    --p-color-sentiment-negative: #D92D21;
+    --p-color-sentiment-positive-hue: 142deg;
+    --p-color-sentiment-positive-saturation: 71%;
+    --p-color-sentiment-positive-lightness: 45%;
```

Comment:
do we have guidelines/preference/opinions on color format? Hex is pretty much out-of-the-question here. I see some mostly hex and some rgba in this file. iirc rgb w/ the space delimited values and slash for alpha channel is the more modern take over rgba? And personally I find HSL nicer to work w/. 

Happy to swap over to RGB/RGBA though for consistency I suppose.